pkg/sloop/server/internal/config/config.go

```diff
@@ -171,17 +172,27 @@ func (c *SloopConfig) Validate() error {
 	return nil
 }
 
-func loadFromFile(filename string) *SloopConfig {
-	yamlFile, err := ioutil.ReadFile(filename)
+func loadFromFile(filename string) (*SloopConfig, error) {
```

Comment:
if we are going to use panic, this error seems only return nil, we can just return *SloopConfig

Reply:
+1 to above, this function never returns error, let's update the signature and return *SloopConfig only.